docs/getting-started/features/tamper-detection.en.md

```diff
@@ -9,76 +9,62 @@ To secure your Krux device, always verify firmware authenticity before installat
 
-- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all four words.
+- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all sets of words.
 
-### Filling Empty Flash Blocks
+### Filling Empty Flash Memory Blocks
 
-Krux performs a memory sweep while simultaneously capturing a live feed from the camera. Whenever an empty block is found in the flash memory, Krux estimates the image's entropy by evaluating its color variance. Krux waits until minimum threshold is met, then uses the data from the image to fill these empty spaces with rich, random entropy.
+Use this to enhance tamper detection. Krux performs a memory sweep while capturing a live feed from the camera. Whenever an empty block is found in the flash memory, it uses the data from the image to fill these empty spaces with rich, random entropy. It estimates the image's entropy by evaluating its color variance waiting until a minimum threshold is met. 
 
 ### Ensuring Tamper Detection
 
-The TC Flash Hash function securely hashes the combination of the *TC Code*, UID, and flash content:
-
-`hash(TC Code,UID,Flash content)` -> Image + Words
-
-Hash properties ensure that without knowing the *TC Code*, UID, and flash content, an attacker cannot reproduce the TC Flash Hash results.
+The *TC Flash Hash* function securely hashes the combination of the *TC Code*, device's UID, and flash memory contents. The hash properties ensure that without knowing these three elements, an attacker will not be able to reproduce the *TC Flash Hash* results.
 
 ## Executing *TC Flash Hash*
 
 After setting a *TC Code* user can use the *TC Flash Hash* feature, available in `Tools -> Flash Tools -> TC Flash Hash`.
 
-By navigating to `Settings -> Security -> TC Flash Hash at Boot`, you can set Krux to always require *TC Flash Hash* verification after device is turned on.
-
-If a wrong *TC Code* is typed at boot, the device will turn off. As storing code typing attempts count on flash would change its contents, there will be no consequences if wrong *TC Code* is typed multiple times.
-
-As *TC Code* verification data is stored in the user's region of memory, *TC Flash Hash* and *TC Code* requirement is disabled if the user wipes the device. Flashing an older firmware version will also disable the feature.
+By navigating to `Settings -> Security -> TC Flash Hash at Boot`, users can set Krux to always require *TC Flash Hash* verification after device is turned on. If a wrong *TC Code* is typed at boot, the device will turn off. Nothing else will happen if the wrong *TC Code* is entered multiple times. As *TC Code* verification data is stored in the user's region of memory, the requirement to type at boot is disabled if the user [erases user's data](../features/tools.md/#erase-users-data) or [wipe device](../installing/from-gui/usage.md/#wipe-device). Flashing an older firmware version will also disable this feature.
```

Comment:
Why make it a single paragraph?

Reply:
It's the same topic or idea. If I change the paragraph, it would indicate a change of emphasis or tone within a topic or it would also give the users a break in their thinking. I believe that with just one paragraph we improve the understanding of the topic.